### PR TITLE
release: v1.36.0

### PR DESCRIPTION
**Pull request by MarlonGamez**
**Description**
Release PR for skaffold v1.36.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,123 @@
...
+Note: This release comes with a new config version, `v2beta27`. To upgrade your skaffold.yaml, use `skaffold fix`. If you choose not to upgrade, skaffold will auto-upgrade as best as it can.
+
+New Features and Additions:
+* feat(logs): allow user config for parsing json application logs when using kubernetes deployers [#7074](https://github.com/GoogleContainerTools/skaffold/pull/7074)
+* feat(kustomize): Template `paths` [#7049](https://github.com/GoogleContainerTools/skaffold/pull/7049)
+* feat: add 'branches' git tag variant [#7006](https://github.com/GoogleContainerTools/skaffold/pull/7006)
+* feat(tag): validate generated/provided tag [#7042](https://github.com/GoogleContainerTools/skaffold/pull/7042)
+* feat: fetch default-repo from local cluster [#6985](https://github.com/GoogleContainerTools/skaffold/pull/6985)
+* feat: allow specifying hook command directory [#6982](https://github.com/GoogleContainerTools/skaffold/pull/6982)
+* feat: monitor OS vulnerability in Skaffold LTS images. [#6964](https://github.com/GoogleContainerTools/skaffold/pull/6964)
```
Comment (tejal29) [inline CHANGELOG.md:29]: should we remove this from features?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,123 @@
...
+`curl -Lo skaffold https://storage.googleapis.com/skaffold/releases/v1.36.0/skaffold-darwin-arm64 && chmod +x skaffold && sudo mv skaffold /usr/local/bin`
+
+**Windows**
+https://storage.googleapis.com/skaffold/releases/v1.36.0/skaffold-windows-amd64.exe
+
+**Docker image**
+`gcr.io/k8s-skaffold/skaffold:v1.36.0`
+
+Note: This release comes with a new config version, `v2beta27`. To upgrade your skaffold.yaml, use `skaffold fix`. If you choose not to upgrade, skaffold will auto-upgrade as best as it can.
+
```
Comment (tejal29) [inline CHANGELOG.md:21]: Should we add a section "highlights" and then talk about Cross platforms builds?
We could link the doc @gsquared94 added in #7090

They should show up at once merged

https://skaffold.dev/docs/pipeline-stages/builders/#cross-platform-build-support

**Review by tejal29 — COMMENTED** [review]
few nits.

File: CHANGELOG.md (hunk 2 above)
Comment (MarlonGamez) [reply CHANGELOG.md:21]: I spoke to Gaurav about this and I believe that we wanted to wait a bit and announce it in a later release once he had fixed/added a few things

File: CHANGELOG.md (hunk 1 above)
Comment (MarlonGamez) [reply CHANGELOG.md]: done.

File: CHANGELOG.md (hunk 2 above)
Comment (tejal29) [reply CHANGELOG.md:21]: sounds good.

**Review by tejal29 — APPROVED** [review]

**Pull request merged**